src/en/getting-started.md

```diff
@@ -49,52 +57,40 @@ particular cloud provider. This is done by generating and editing the following
-
-    juju generate-config
+Juju can automatically generate this file in this way:
 
-This command will cause a file to be written to your ~/.juju directory if an
-environments.yaml file does not already exist. It will also create the ~./juju
-directory if that does not exist.
-
-This file will contain sample profiles for different types of cloud services,
-but you will need to edit the files to provide specific information for your
-cloud provider. Sections are created for Amazon (AWS) services, HPCloud and a
-generic OpenStack instance (and for the local provider if running on Linux). For
-more specifics on what needs to be changed, see the relevant sections below.
+```bash
+juju generate-config
+```
 
-  - [Configuring for Amazon AWS](config-aws.html)
-  - [Configuring for Windows Azure](config-azure.html)
-  - [Configuring for HP Cloud](config-hpcloud.html)
-  - [Configuring for OpenStack](config-openstack.html)
-  - [Configuring for bare metal using MAAS](config-maas.html)
-  - [Configuring for LXC local provider (Linux)](config-local.html)
-  - [Configuring for Vagrant local provider (OSX, Linux, and Windows)](config-vagrant.html)
+This action will not overwrite an existing file but merely dump the information
+onscreen (STDOUT). It will contain sample profiles for different types of cloud
+services.  Edit it to provide specific information for your chosen cloud
+provider. For more specifics on what needs to be changed, see the relevant
+sections in the left pane (under *Install & Configure*).
 
-**Note:** Juju's command line interface includes documentation, doing `juju help` will show you the topics. You can also check out the [Juju command cheatsheet](https://github.com/juju/cheatsheet) if you are looking for a handy command guide.
+**Note:** Juju's command line interface includes documentation, doing `juju
+help` will show you the topics. You can also look at the [Juju command
+cheatsheet](https://github.com/juju/cheatsheet) if you are looking for a handy
+command guide.
 
 # Testing your setup
 
-Once you have installed and configured Juju, it is probably a good idea to take
-it for a bit of a test drive and check that everything is working as expected.
-Because Juju makes it really easy to deploy services, this is actually quick and
-straightforward.
-
-The first thing to do is set up a bootstrap environment. This is an instance in
-the cloud that Juju will use to deploy and control other services with. It will
-be created according to the configuration you have provided, and your SSH key
-will automatically be uploaded so that Juju can communicate securely with the
-bootstrap instance.
+The first step is to create a bootstrap environment. This is a cloud instance
+that Juju will use to deploy and manage services. It will be created according
+to the configuration you have provided, and your SSH key will automatically be
+injected so that Juju can communicate securely with the bootstrap instance.
```

Comment:
I understand. I think we want to worry less about being concise and more about using natural language which expresses things clearly, especially in the pages aimed at less experienced users. It is certainly okay to cut out unnecessarily verbose passages, but I think you might want to rein that in a little bit when it reduces the conversational nature of introductory text. Indeed, 'injected' is a word that is commonly used in tech docs around this subject, but "uploaded" is actually what happens, is arguably easier to understand, and certainly easier to translate.